easycv/pipeline.py

```diff
@@ -42,6 +48,57 @@ def __init__(self, source, name=None):
         else:
             raise InvalidPipelineInputSource()
 
+    @staticmethod
+    def _calculate_forwards(source):
+        outputs = {}
+        forwards = {}
+
+        for i in range(len(source)):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider using enumerate instead of iterating with range and len](https://app.codacy.com/gh/easycv/easycv/pullRequest?prid=5625158)